typescript/packages/well-log-viewer/src/WellLogViewer.stories.tsx

```diff
@@ -3,7 +3,49 @@
 import React from "react";
 import WellLogViewer from "./WellLogViewer";
 import { argTypesWellLogViewerProp } from "./WellLogViewer";
-import { colorTables } from "@emerson-eps/color-tables";
+
+import exampleData from "../../../../example-data/deckgl-map.json";
+
+import { WeakValidationMap } from "react";
+import SubsurfaceViewer from "@webviz/subsurface-viewer";
```

Comment:
Wow, I'm really not in favor of finding a dependency on subsurface-viewer here :/
Any chance to consider moving all the storybook stuff in its own package ? Storybook is anyhow built on top level-directory ?

Reply:
This doesn't add any dependency, so I don't understand the issue.

Reply:
This is only because we took care to exclude stories from build (was not always the case ;) )
I think it would express clearer code organization, and avoid people looking at the code to have to think twice about it before understanding that it does not reflect a dependency...
I can do it in a separate PR.

Reply:
Let's do it in a separate PR...